joint_control:
  joint_limits:
    joint2:
      has_position_limits: true # false for continuous joints
      min_position: -1.0
      max_position: 1.0
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: true
      max_acceleration: 5.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 25.0

  pid_gains:
    joint1: {p: 200, i: 0.0, d: 0.1}
    joint2: {p: 200, i: 0.0, d: 0.1}